alias/dlgalias.c

```diff
@@ -198,20 +207,36 @@ static void dlg_select_alias(char *buf, size_t buflen, struct AliasMenuData *mda
   menu->make_entry = alias_make_entry;
   menu->custom_search = true;
   menu->tag = alias_tag;
-  menu->title = _("Aliases");
-  menu->max = ARRAY_SIZE(mdata);
+  menu->max = menu_count_visible(&mdata->alias_arr);
   menu->mdata = mdata;
 
+  if (mdata->str)
+  {
+    char *tmp_str = NULL;
+    char *new_title = NULL;
+
+    mutt_str_asprintf(&tmp_str, _("Limit: %s"), mdata->str);
+    mutt_str_asprintf(&new_title, _("Aliases - %s"), tmp_str);
+
+    FREE(&tmp_str);
+
+    menu->title = new_title;
+  }
+  else
+  {
+    menu->title = strdup(_("Aliases"));
+  }
```

Comment:
This block of code is repeated in `pattern/pattern.c`
I suggest a adding small function taking `(AliasMenuData *mdata, char *limit)` where `limit` could be `NULL`.

Also, the string `"Aliases"` is already translated, but `"Aliases - %s"` isn't.
If we can use the first one _creatively_, then we save the translators a job.